Kudu.Core/Deployment/DeploymentManager.cs

```diff
@@ -544,8 +544,8 @@ private DeployResult GetResult(string id, string activeDeploymentId, bool isDepl
                 string repositoryRoot = _environment.RepositoryPath;
                 var perDeploymentSettings = DeploymentSettingsManager.BuildPerDeploymentSettingsManager(repositoryRoot, _settings);
 
-                string delayMaxInStr = perDeploymentSettings.GetValue(SettingsKeys.MaxRandomDelayInSec, false);
-                if (delayMaxInStr != null)
+                string delayMaxInStr = perDeploymentSettings.GetValue(SettingsKeys.MaxRandomDelayInSec, true);
```

Comment:
I was reading its implementation, it appears that if I put `true`, less conditions and less code will run, but since I don't really care about its value, I am going back to default